### DISCUSSION TITLE
Improvements to windows exe

### DISCUSSION THREAD

**Discussion by klepp0906** [FlareSolverr/FlareSolverr#756] [Ideas]
So at current if we want to run flaresolverr without needing to have a terminal/console window open in perpetuity we have to set it up to run as a scheduled task. 

would be great to be able to put the exe in the shell:startup folder but with an argument to start it sans the console window.  --noconsole or some such, as most of the time you'd want this running in the background. 

a further improvement yet would be a tray icon so could simply use that to see the console/output if they wanted instead of having to choose as there are also times youd want to see the output and ending/restarting the process via a different means to do so is cumbersome. 

just some thoughts.  windows users are the worst :P

**Comment by ngosang** [comment]
PRs are welcome.

**Reply by klepp0906** [reply]
oh trust that if i could (code) i wouldnt have placed this in the ideas section, i would have just gotten to it ;p  unfortunately just a user, and as such at the mercy of you generous folks <3

**Reply by Urist0595** [reply]
@ngosang This is just completely user-hostile. Not only are there zero instructions for how to set this up on Windows (had to figure it on my own/with the help of the Jackett wiki) the response to someone asking for a basic feature like being able to minimize the application window to the tray and not having to deal with having a console window open perpetually is "add the feature yourself". So because some of us can't use Linux, we just have to deal with a gimped Windows version that doesn't have the same features?

**Reply by ilike2burnthing** [reply]
The basic script and instructions have been available here for the last 2.5yrs - https://github.com/FlareSolverr/FlareSolverr/issues/70#issuecomment-761954514

You've managed to read malice from this comment that the OP and I didn't, so I'd suggest the issue is in your interpretation.

**Reply by Urist0595** [reply]
@ilike2burnthing Thank you. I've tested this with the application, and it appears to work. 

I interpreted malice from the reply because I've seen that kind of malice toward any Windows user asking for features on the Windows version of certain programs made by and originally for Linux users. I admit that I made a mistake in this case by assuming that was the intent here behind Ngosang's reply, and I apologize for that.

That being said, that script file appears to work perfectly well. At least some mention of it on the readme would be very useful instead of leaving it buried in the issues for 2.5 years as you said it was. I had no idea that script existed until you replied with it. Better yet, including the script in the Windows binary itself would be even better.

In any case, thanks, and apologies again.

**Comment by qinguoluo** [comment]
How to install flaresolverr on windows and configure it to work with jackett? There are not enough information on internet.

**Reply by ilike2burnthing** [reply]
https://github.com/FlareSolverr/FlareSolverr#precompiled-binaries

https://github.com/Jackett/Jackett#configuring-flaresolverr
